test/integration/kubernetes_deploy_test.rb

```diff
@@ -1593,6 +1592,25 @@ def test_fail_erb_templates_if_rendering_is_disabled
     ], in_order: true)
   end
 
+  def test_deploy_allow_globals_warns
+    result = deploy_fixtures("globals")
```

Comment:
Don't we already have a fixture set with a global resource in it (CRD or storage class probably?)?

Reply:
We do, my thought was this would be the start of the `hello-cloud` but for global resources. I'm fine to save that for the first person to actually add real code to a global resource.

Reply:
True, we're going to have to separate that out entirely, so a purpose-built fixture set makes sense.